schema/openeox_core_json_schema.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://docs.oasis-open.org/openeox/tbd/openeox_core_json_schema.json",
+  "title": "EOL and EOS Information",
+  "description": "A draft schema for representing End-of-Life (EOL) and End-of-Support (EOS) information in OpenEoX.",
+  "type": "object",
+  "$defs": {
+    "schema_t": {
+      "title": "OpenEoX Core schema",
+      "description": "Specifies the schema the JSON object must be valid against.",
+      "type": "string",
+      "enum": [
+        "https://docs.oasis-open.org/openeox/tbd/openeox_core_json_schema.json"
```

Comment:
"Same" here: `https://docs.oasis-open.org/openeox/tbd/schema/openeox-core.json`

Reply:
addressed in [b57311f](https://github.com/oasis-tcs/openeox/pull/10/commits/b57311f3895a5208794ebec25f72fbf200fb4d3e)